CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to theupdateframework.io
```

Comment:
@Ayush9026 you did a wonderful work, I've just a small suggestion, should we add testing guidelines too, on how to test changes locally same as [python-tuf CONTRIBUTING guideline.](https://theupdateframework.readthedocs.io/en/latest/CONTRIBUTING.html). Steps can be this:


-  Execute the following command to install the project's dependencies locally:
   ` $ yarn install`
    
-  Start the Hugo development server to preview the website locally by running:
   `$ hugo server`
  
-   Once started, the website will be accessible at: http://localhost:1313/    by default. And review your changes.

Reply:
Hi @h4l0gen,

Thank you for the kind words! 😊 That's a great suggestion. Adding testing guidelines for local changes will definitely help contributors. I'll include the steps you mentioned once @lukpueh and @joshuagl review it.

Thanks again for your valuable feedback! 🙌